packages/extension/src/lib/extensionScripts.ts

```diff
@@ -20,7 +20,7 @@ export const HOST_PERMISSIONS = [
   'https://*.daily.dev/*',
 ];
 
-const hasInjectedScripts = false;
+let hasInjectedScripts = false;
```

Comment:
This even looks like it was meant to do something like this? 🤔

Reply:
Yeah. It did it originally
https://github.com/dailydotdev/apps/blob/111feb347c5cd312a6cc6d75a54043005dbd5a3c/packages/extension/src/lib/extensionScripts.ts#L26-L35

Reply:
Probably lint auto-fix that saw a let that wasn't re-written and converted it to a const. That's what happens locally with me if I remove the `hasInjectedScripts = true;`, then it's convert to `const hasInjectedScripts = false;`.

Reply:
And then no-one caught the change